.circleci/config.yml

```diff
@@ -72,6 +93,11 @@ jobs:
           > version.json
       - run: docker build --pull -t addons-frontend .
       - run: docker images
+      - run:
+          name: Test Image
+          command: |
+            docker run --net "host" --detach addons-frontend yarn amo:dev
```

Comment:
Is there a way in yaml to reference this so it's not copy/pasted from above? I'm worried that the two commands could go out of sync.